Gulpfile.js

```diff
@@ -100,13 +100,12 @@ function getBabelBuildPipeline() {
 gulp.task('css:library', function() {
   return merge(
     gulp.src([
-      './bower_components/angular/angular-csp.css',
-      './bower_components/angular-loading-bar/build/loading-bar.min.css',
-      './bower_components/angular-motion/dist/angular-motion.min.css',
+      './node_modules/@bower_components/angular/angular-csp.css',
+      './node_modules/@bower_components/angular-loading-bar/build/loading-bar.min.css',
+      './node_modules/@bower_components/angular-motion/dist/angular-motion.min.css',
       './node_modules/font-awesome/css/font-awesome.min.css',
-      './bower_components/c3/c3.min.css',
```

Comment:
seems c3.min.css is removed, is this intentional? If so, are we sure that it is not used anywhere in the code?

Reply:
yes, we have checked the codebase and found that the `c3-` css classes are not being used anywhere.